leshan-core/src/main/java/org/eclipse/leshan/core/node/codec/json/LwM2mNodeJsonEncoder.java

```diff
@@ -174,6 +175,11 @@ public void visit(LwM2mResource resource) {
             resourceList = lwM2mResourceToJsonArrayEntry(null, timestamp, resource);
         }
 
+        @Override
+        public void visit(LwM2mResourceInstance instance) {
+            throw new UnsupportedOperationException("not yet implemented");
```

Comment:
hi @sbernard31 , are we suppose to add implementation here ? rht?

Reply:
If you want to add support of  READ request for **resources instances operations** with LWM2M 1.0 Json format, this is here yes.

Reply:
how you identify that these changes leads to LWM2M 1.0 Json format support, for  LWM2M 1.1 Json format is deprecated OR some other java file responsible for it?

Reply:
correct this format is deprecated in LWM2M 1.1 (but we need to support it for backward compatibility)

The new json format is senML-json but currently not finished to be implemented look at `LwM2mNodeSenMLJsonEncoder`. (and some orphan PR : #68, #708)

What are your needs about "resources instances operations", I mean the priority for you ? which request, which format ?

Reply:
For my project as Of now I'm interesting in TEXT and TLV formats for READ and WRITE resource instance operations. Maybe in future we will use also senML+json and cbor , but not now. Im totally fine with JSON implementation for backward compatibility.

Reply:
Ok so let's focus  on TEXT/TLV and READ/WRITE.

TLV content format : it's done :heavy_check_mark:  (see e23e000)
`ReadRequest` it's done :heavy_check_mark:  (commit 3800870 and 546d3bd)

So my advice : 
- update `WriteRequest`  API to be able to support resource instance operation. (inspired yourself with 3800870)
- update Leshan client code too ( inspired by 546d3bd)
- then add support to Text content format (look at `LwM2mNodeTextDecoder` and `LwM2mNodeTextEncoder`) (inspired by e23e000)
- then you should add some integration tests, look at `WriteTest` in _leshan-integration-tests_. You should be able to write a read/write test easily. 

For old json, I'm not even sure we need to add it, so let's this aside.
For senML+json and cbor, we need to write the encoder/decoder (resource instance operation will probably comes naturally with those implementation).

Reply:
Thanks for help. Im already almost done with Json, so I will commit what I have and will take some insight from you about commit.

Reply:
I'm trying to push the changes:
`Push failed: Failed with error: unable to access 'https://github.com/eclipse/leshan.git/': The requested URL returned error: 403`. Do I have permissions to do it?

Reply:
The error is "normal". 
=> Look at https://github.com/eclipse/leshan/wiki/How-to-contribute (especially the **Get Ready to Code !**), this should help.

We need to support java 1.7, do you face any issue about this ?

Reply:
Looking to it... 
I mean I tried introduce some code features from java 8 as static methods in interfaces but if we should support 1.7 as well I can't do it probably.

Reply:
@sbernard31 , any update on my questions?

Reply:
I'm not sure which one I didn't answer ? :sweat_smile: 
About signing, look at : https://github.com/eclipse/leshan/pull/882#issuecomment-682502985
About java 8, unfortunately for now we target 1.7 as minimal version so you should avoid to use java8 stuff. 

Or maybe I missed something ?

Reply:
Sorry for confusion, the question regarding sing off commits, could you please look in to it. It said footer is not there but it is, what can be missing more that check failed?

Reply:
nothing missing, thanks for comments. I just missed the comments in PR itself )